libs/community/langchain_community/llms/yandex.py

```diff
@@ -104,6 +107,13 @@ def validate_environment(cls, values: Dict) -> Dict:
             values[
                 "model_uri"
             ] = f"gpt://{values['folder_id']}/{values['model_name']}/{values['model_version']}"
+        if values["disable_request_logging"]:
+            values["_grpc_metadata"].append(
```

Comment:
pardon. this change causes the following problem:
If I don't provide `iam_token` line 101 creates `((,),)` instead of `[(,),]` then disabling logging falls with:

```
File ~/git/...../python3.11/site-packages/langchain_community/embeddings/yandex.py:119, in YandexGPTEmbeddings.validate_environment(cls, values)
...

AttributeError: 'tuple' object has no attribute 'append'
```

I suppose `_grpc_metadata` should be a list always. May I come up with this typo fix PR?

Reply:
Thanks, you're welcome to fix this bug.

Reply:
for the record (TBC)
passing just api_key and folder_id fails with:
```
File ~/git/legal_comply/.conda/lib/python3.11/site-packages/langchain_community/embeddings/yandex.py:211, in _make_request(self, texts, **kwargs)
..
--> 211 res = stub.TextEmbedding(request, metadata=self._grpc_metadata)  # type: ignore[attr-defined]

AttributeError: 'YandexGPTEmbeddings' object has no attribute '_grpc_metadata'
```

Reply:
It seems like something have changed in pydantic. After `values["_grpc_metadata"]` is set it goes nowhere since this field is private.

Reply:
Ok. I've managed to work with `api_key=,folder_id=,disable_request_logging=` copying and hacking it locally
- changing tuple to list
- dropping leading underscore from `_grpc_metadata` -> `grpc_metadata`. 
Summon more _pydantic_ people to discuss a proper fix.

Reply:
> Thanks, you're welcome to fix this bug.

#25617 done